docs/quickstart.md

```diff
@@ -0,0 +1,765 @@
+## Structured Logging
+In the next step, you have been given the task of proposing production quality logging capabilities to your lambda code.
+We want our log event to be in a JSON format. Also, You decided to follow [structured logging approach](https://docs.aws.amazon.com/lambda/latest/operatorguide/parse-logs.html). In a result, we expect easy to search, consistent logs containing enough context and data to analyse the status of our system. We can take advantage of CloudWatch Logs and Cloudwatch Insight for this purpose.
+
+The first option could be to use a python logger in combination with the `pythonjsonlogger` library for simple structured logging.
+
+=== "app.py"
+
+    ```python hl_lines="1 4 8-13 20 26 31"
+    import logging
+    import os
+
+    from pythonjsonlogger import jsonlogger
+
+    from aws_lambda_powertools.event_handler.api_gateway import ApiGatewayResolver
+
+    logger = logging.getLogger("APP")
+    logHandler = logging.StreamHandler()
+    formatter = jsonlogger.JsonFormatter(fmt="%(asctime)s %(levelname)s %(name)s %(message)s")
+    logHandler.setFormatter(formatter)
+    logger.addHandler(logHandler)
+    logger.setLevel(os.getenv("LOG_LEVEL", "INFO"))
+
+    app = ApiGatewayResolver()
+
+
+    @app.get("/hello/<name>")
+    def hello_name(name):
+        logger.info(f"Request from {name} received")
+        return {"statusCode": 200, "body": {"message": f"hello {name}!"}}
+
+
+    @app.get("/hello")
+    def hello():
+        logger.info(f"Request from unknown received")
+        return {"statusCode": 200, "body": {"message": "hello unknown!"}}
+
+
+    def lambda_handler(event, context):
+        logger.debug(event)
+        return app.resolve(event, context)
+    ```
+=== "requirements.txt"
+
+    ```bash
+    aws-lambda-powertools
+    python-json-logger
+    ```
+On the first try, we did a couple of steps to set up our logging:
+
+* Create an application logger called `APP`.
+* Configure handler and formatter.
+* Set log level.
+
+After that, we use this logger in our application code to record the required information. We see logs structured as follows:
+```json
+{"asctime": "2021-11-22 15:32:02,145", "levelname": "INFO", "name": "APP", "message": "Request from unknown received"}
+```
+instead of
+```json
+[INFO]  2021-11-22T15:32:02.145Z        ba3bea3d-fe3a-45db-a2ce-72e813d55b91    Request from unknown received
+```
+
+So far, so good! To make things easier, we want to add extra context to the logs. We can extract it from a lambda context or an event passed to lambda handler at the time of invocation. We have to make sure that we always add those specific attributes wherever a logger is used. Can we ensure that the required attributes are added automatically on our behalf without having to move them around? Yes! Powertools Logger to the rescue :-)
```

Comment:
We have to make sure that we always -> We need

Reply:
fixed